erigon-lib/downloader/torrent_files.go

```diff
@@ -233,6 +229,9 @@ func (tf *AtomicTorrentFS) newDownloadsAreProhibited(name string) (bool, error)
 	if err != nil {
 		return false, fmt.Errorf("NewDownloadsAreProhibited: read file: %w", err)
 	}
+	if exists && len(torrentListJsonBytes) == 0 { // backward compatibility: if .lock exists and empty - it means everything is prohibited
```

Comment:
Maybe just mark all snapshots as prohibited except for blobsidecars

Reply:
I have idea to make .lock also forward compatible - by making it whitelist instead of blacklist: after adding new snap type it will not be downloaded by accident. Will do it in next PR. (or you can)